File: 1368/1368.cs
public class Solution {
    public int MinimumCostToMakeAtLeastOneVal(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumCostToMakeAtLeastOneVal(root.left), MinimumCostToMakeAtLeastOneVal(root.right));
    }
}

File: 1368/1368.go
func minimumCostToMakeAtLeastOneVal(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumCostToMakeAtLeastOneVal(root.Left)
	r := minimumCostToMakeAtLeastOneVal(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}